torch/csrc/jit/interpreter.cpp

```diff
@@ -735,6 +756,11 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
     if (runImpl(stack)) {
       future->wait();
 
+      if (future->hasError()) {
+        auto& error = future->getError();
+        throwError(error.error_msg, error.is_jit_exception);
```

Comment:
Seems dangerous. If error_msg is blank, then this will entry  a `throw;` pathway but it is not in an error state. Seems better to just rely on `future.value()` throwing an exception.